addons/website/tests/test_theme.py

```diff
@@ -30,7 +30,7 @@ def _get_header_template_key():
         self.assertEqual(_get_header_template_key(), 'website.template_header_default',
                          "Only the default template should be active.")
 
-        key = 'website.template_header_magazine'
+        key = 'website.template_header_stretch'
```

Comment:
This seems to be using a template that is not defined yet at this point in the github history + magazine does not exist anymore at this point either.

Reply:
Good catch ! I changed it with the `vertical` template